extras/ffmpeg-decoder/src/ffmpeg/avcodec.rs

```diff
@@ -747,6 +752,10 @@ pub struct AvPacket<'a> {
     _buffer_data: PhantomData<&'a ()>,
 }
 
+/// SAFETY: safe because data in 'AvPacket' is valid, and thus safe to send its reference.
+unsafe impl<'a> Send for AvPacket<'a> {}
+unsafe impl<'a> Sync for AvPacket<'a> {}
```

Comment:
As for `AvCodec`, `AvPacket` does not allow mutation, so it should be safe to  implement `Sync` on it?